.mvn/gradle-enterprise-custom-user-data.groovy

```diff
@@ -12,4 +12,4 @@ buildScan.tag(project.name)
 buildScan.value('parallel', session.parallel as String)
 
 // Enable storing in the remote build cache based on the presence of a `CI` environment variable
-buildCache.remote.storeEnabled = session.systemProperties.containsKey('CI')
+buildCache.remote.storeEnabled = System.getenv('GITHUB_ACTIONS') != null
```

Comment:
The comment does not match anymore.

Given this script file is only a sample to show how to use the API, and that we do not really want to be applied when running the build here, I suggest we set another, less impactful (neutral) property on the buildCache API.

Reply:
I changed it to set storing in the local build cache. How does it look?

Reply:
Good idea. LGTM.